types/ember__string/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Type definitions for @ember/string 3.0
-// Project: https://emberjs.com/api/ember/3.4/modules/@ember%2Fstring
```

Comment:
At least for now, https://github.com/emberjs/ember-string is a different package, and I cannot guarantee that the `@ember/string` types align with anything other than 

https://emberjs.com/api/ember/3.4/modules/@ember%2Fstring

Reply:
Sounds like ember__string should be marked the same as the other `@ember/*` packages then. I'll include it in the non-npm packages PR.